status-handler.js

```diff
@@ -297,17 +221,25 @@ class StatusHandler {
 
     switch (event) {
       case BUILD_COMPLETED:
-        if (!await this._isBuildCompleted(data)) return;
         body = { event, pkg, version, env };
         break;
       case BUILD_STARTED:
-        if (!await this._isBuildQueued(data)) return;
+        if (!this._isBuildQueued(data)) return;
         body = { event, pkg, version, env };
         break;
       default:
         throw new Error(`'${event}' is not a valid webhook type`);
     }
 
+    const repository = await new Promise((resolve, reject) => {
+      this.wrhs.packages.get({ pkg, version, env }, (err, result) => {
```

Comment:
only `pkg` name matters, here. Like doing a npm view

Reply:
I want the version as well for sure, coz the repo url may be updated with the version. It may be a very rare case but it costs nothing here to ensure the version as well.

Reply:
Discussed offline. Makes sense. Changing this.